app/code/Magento/CatalogGraphQl/Model/Resolver/Product/ProductImage/Path.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\CatalogGraphQl\Model\Resolver\Product\ProductImage;
+
+use Magento\Catalog\Model\Product;
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\GraphQl\Config\Element\Field;
+use Magento\Framework\GraphQl\Query\ResolverInterface;
+use Magento\Framework\GraphQl\Schema\Type\ResolveInfo;
+
+/**
+ * Returns product's image path
+ */
+class Path implements ResolverInterface
```

Comment:
No need to have this resolver, its logic can be moved to the main image resolver to gain better performance.

Reply:
Removed